src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -205,8 +267,8 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
                         COMPLETION OF DELIVERIES
                     </Text>
                 </View>
-                <View style={[styles.flexColumn, { width: "100%" }]}>{header}</View>
-                <View style={[styles.tableSection, styles.flexColumn]}>{table}</View>
+                {deliveries.length ? deliveriesTable : null}
+                {collections.length ? collectionsTable : null}
```

Comment:
Think this is the standard way of  doing optional components in react?
```suggestion
                {deliveries.length ?? deliveriesTable}
                {collections.length ?? collectionsTable}
```